model/database/metrics.yaml

```
@@ -133,3 +134,39 @@ groups:
     attributes:
       - ref: db.client.connection.pool.name
         requirement_level: required
+
+  - id: metric.db.cosmos.operation.request_charge
```

Let's be consistent with `cosmosdb` everywhere

```suggestion
  - id: metric.db.cosmosdb.operation.request_charge
```